### PR TITLE
fix(tabs): replace tab character by space characters

### PR DESCRIPTION
**Pull request by benoitf**
#### What it does
just replace tab characters by space

#### How to test
you may reproduce this fix just by opening and saving the file.
About test, hard to say how to test as it's like a no-op but plug-ins should still work as usual
And in github diff view, you can see it's only related to spaces/indent

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Review by vince-fugnitto — APPROVED** [review]
The changes look good to me :+1: 

@benoitf what do you think of enabling the following eslint rule: https://eslint.org/docs/rules/indent
We can enforce that `spaces` are used always, and display warnings or errors when `tabs` are used.

**Comment by benoitf** [comment]
@vince-fugnitto yes would be nice because when we just open/save files, they're producing changes
so a rule would help 👍

File: packages/plugin/src/theia.d.ts [hunk 1]
```diff
@@ -1582,168 +1582,168 @@ declare module '@theia/plugin' {
         waitUntil(thenable: PromiseLike<any>): void;
```
Comment (vince-fugnitto) [inline packages/plugin/src/theia.d.ts:1582]: I just wanted to mention that this file is never checked by `eslint` due to the following `ignorePatterns`:

https://github.com/eclipse-theia/theia/blob/40b380062f5f9976ba46b38db7c7fe2fa56e9535/configs/base.eslintrc.json#L21-L24

Comment (benoitf) [reply packages/plugin/src/theia.d.ts:1582]: maybe this one should not be excluded as it's maintained by us manually

Comment (vince-fugnitto) [reply packages/plugin/src/theia.d.ts:1582]: > maybe this one should not be excluded as it's maintained by us manually

I think so, we should update the `ignorePatterns` to only check `*d.ts` files located under `lib` folders correct?

**Comment by vince-fugnitto** [comment]
@benoitf would you like to merge, I'll merge my pull-request #8630 right after and fix the `base.eslintrc.json` to exclude `lib` folders only and not all `*.d.ts` files.

**Pull request merged**